android/build.gradle

```diff
@@ -13,7 +13,7 @@ import groovy.json.JsonSlurper
 //   - https://github.com/facebook/react-native/blob/0.58-stable/local-cli/templates/HelloWorld/android/app/build.gradle
 
 def DEFAULT_COMPILE_SDK_VERSION = 34
-def DEFAULT_MIN_SDK_VERSION = 19
+def DEFAULT_MIN_SDK_VERSION = 24
```

Comment:
min SDK version is updated to 26 from Point SDK v17.0.0. So this should be updated here.